### ISSUE TITLE
readme

### ISSUE THREAD

**Issue by benibela** [qt4cg/qtspecs#17]
The readme needs an update

Link to the 4.0 html spec files rather than the 3.1 spec

**Comment by martin-honnen** [comment]
I agree but the specs are also now browsable from https://qt4cg.org/ so perhaps the readme should point to that entrance as well.

**Comment by benibela** [comment]
> I agree but the specs are also now browsable from https://qt4cg.org/ so perhaps the readme should point to that entrance as well.

and that should show the `-diff` versions or it is hard to find the changes

**Comment by cedporter** [comment]
Better late than never, the readme is updated including links to the latest specs as HTML and the diffs.
